pkg/balancer/defaultAnalyzer.go

```diff
@@ -47,23 +55,85 @@ func (a *DefaultAnalyzer) AnalyzeLoads() ([]schema.Schedule, bool) {
 	return newSchedules, len(newSchedules) > 0
 }
 
-func (a *DefaultAnalyzer) detectImbalanceShards(loads []*ShardMetrics) ([]uint64, error) {
+func (a *DefaultAnalyzer) detectImbalanceShards(loads []*NodeMetrics) ([]uint64, error) {
 	var imbalancedShards []uint64
+	avgEntries := 0.0
 	for _, load := range loads {
-		if load.CPU > 80.0 || load.Memory > 1<<30 { // 1 GiB
-			imbalancedShards = append(imbalancedShards, load.ShardID)
+		avgEntries += float64(load.NumEntries)	
+	}
+	avgEntries /= float64(len(loads))
+	
+	// If a shard is responsible for more than twice the average number of entries, it is considered imbalanced
+	for idx, load := range loads {
+		if float64(load.NumEntries) > avgEntries * 2 {
+			imbalancedShards = append(imbalancedShards, uint64(idx))
 		}
 	}
 	return imbalancedShards, nil
 }
```

Comment:
Is this just a dummy load balancing strategy?

Reply:
Yes